assets/performanceprofile/tuned/openshift-node-performance-amd-x86

```diff
@@ -0,0 +1,23 @@
+[main]
+summary=Platform specific tuning for AMD x86
+
+[variables]
+
+# AMD specific tuning options
+iommu=+amd_iommu=on iommu=pt
+hpc_cstate=+processor.max_cstate=1
+pppm_pstate=+amd_pstate=active
```

Comment:
I would move the variable values directly to the block below OR keep just the variables (with nicer names) and move the IF block back to the main profile for readability.